app/scripts/controllers/metametrics.test.js

```diff
@@ -604,6 +604,51 @@ describe('MetaMetricsController', function () {
     });
   });
 
+  describe('trackEvent', function () {
+    it('should call only metricsController._trackEvent if actionId is not provided', function () {
+      const metaMetricsController = getMetaMetricsController({
+        participateInMetaMetrics: false,
+      });
+      const spyTrackEvent = sinon.spy();
+      metaMetricsController._trackEvent = spyTrackEvent;
+
+      const spyCreateEvent = sinon.spy();
+      metaMetricsController.createEventFragment = spyCreateEvent;
+
+      const spyFinalizeEvent = sinon.spy();
+      metaMetricsController.finalizeEventFragment = spyFinalizeEvent;
+
+      metaMetricsController.trackEvent({}, {});
+
+      assert(spyTrackEvent.calledOnce);
+      assert(spyCreateEvent.notCalled);
+      assert(spyFinalizeEvent.notCalled);
+
+      // spyTrackEvent.
+    });
+
+    it('should call createEventFragment, finalizeEventFragment if actionId is provided', function () {
+      const metaMetricsController = getMetaMetricsController({
+        participateInMetaMetrics: false,
+      });
+      const spyTrackEvent = sinon.spy();
+      metaMetricsController._trackEvent = spyTrackEvent;
+
+      const stubCreateEvent = sinon.stub();
+      stubCreateEvent.returns({ id: 'DUMMY_ID' });
```

Comment:
Super nit:  Could we make `DUMMY_ID` a constant and reference it?

Reply:
fixed